server/migrations/20200521003201-create_table_test_issues.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
             }
         });
     },
-    down: (queryInterface) => {
+    down: queryInterface => {
```

Comment:
Lol, I've been discarding this change all day, every time my linter fixes it, because I wasn't working on anything that would make sense to include this change 🤣